client/src/app/+admin/config/edit-custom-config/edit-custom-config.component.ts

```diff
@@ -137,6 +137,11 @@ export class EditCustomConfigComponent extends FormReactive implements OnInit, A
           css: null
         }
       },
+      federation: {
+        videos: {
+          federateUnlisted: true
```

Comment:
Do you want admins to update this setting on the web interface? If no, you can remove this

Reply:
Is there any reason not to allow this? I added it to `production.yaml.example` as well, based on your earlier comment.

Reply:
> Is there any reason not to allow this?

It subjective, but it's not a setting admins will update frequently. So I don't think it's very useful to put this in the admin interface. But you're the one who does, so you're the one who decides :)

Reply:
Hm, I guess the main benefit would be discoverability.
I'm open to whatever you think - if you feel like it doesn't belong in the web interface, I'm willing to do the rework.

Reply:
Then let's move this out of the web interface. Admins should read the config' file when they install peertube, and we'll mention this in the changelog. You could also mention this setting in https://docs.joinpeertube.org/#/admin-following-instances?id=being-followed-by-an-instance

Reply:
Ok, done